bundle/metadata/annotations.yaml

```diff
@@ -1,7 +1,10 @@
 annotations:
-  operators.operatorframework.io.bundle.channel.default.v1: "4.7"
-  operators.operatorframework.io.bundle.channels.v1: "4.7"
+  operators.operatorframework.io.bundle.channel.default.v1: ""
+  operators.operatorframework.io.bundle.channels.v1: alpha
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.mediatype.v1: registry+v1
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: elasticsearch-operator
+  operators.operatorframework.io.metrics.builder: operator-sdk-unknown
```

Comment:
This looks like the hook to declare a proper sdk version that shows up as an annotation in the CSV.